docs/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+  "extends": "../tsconfig.json",
+  "compilerOptions": {
+    "jsx": "react-jsx",
+    "module": "esnext",
+    "moduleResolution": "bundler",
+    "lib": ["dom", "dom.iterable", "esnext"],
+    "types": [
+      "@docusaurus/module-type-aliases",
+      "@docusaurus/types",
+      "react",
```

Comment:
Can these entries be moved to the tsconfig file in the root directory? The entire code base is typescript.